### PR TITLE
fix video readme

### PR DESCRIPTION
**Pull request by MarcSkovMadsen**
- On laptop the README the video is removed
- In ios github app the `video is not supported` message is shown

The README is a bit *dry* without a picture or video at the top.

This PR fixes this. Preview the README [here](https://github.com/holoviz-topics/panel-chat-examples/tree/fix/video-readme).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -173,3 +173,4 @@ docs/langchain.md
 docs/mistral.md
 docs/openai.md
 docs/examples/langchain/penguins.csv
+docs/index.md
```
Comment (MarcSkovMadsen) [inline .gitignore:176]: The `index.md` file is now generated by the script. I could find no other way to include the video both in the README and the docs.

File: README.md [hunk 2]
```diff
@@ -4,16 +4,11 @@
...
 
-**Documentation**: <a href="https://holoviz-topics.github.io/panel-chat-examples/" target="_blank">https://holoviz-topics.github.io/panel-chat-examples/</a>
+**Documentation**: <a href="https://holoviz-topics.github.io/panel-chat-examples/" target="_blank">holoviz-topics.github.io/panel-chat-examples/</a>
 
-<video controls style="height:auto;width: 100%;max-height:500px">
-    <source src="https://github.com/holoviz-topics/panel-chat-examples/raw/main/assets/videos/panel-chat-examples-splash.mp4" type="video/mp4">
-    Your browser does not support the video tag.
-</video>
+https://github.com/holoviz-topics/panel-chat-examples/assets/42288570/cdb78a39-b98c-44e3-886e-29de6a079bde
```
Comment (MarcSkovMadsen) [inline README.md:9]: If we create a new video later, we will need to drop it into the Github README to get a working url.

**Review by ahuang11 — APPROVED** [review]

**Comment by ahuang11** [comment]
Going to wait to evening to merge this because it takes a bit for the docs to rebuild after deploying and I don't want the posts' links to go to an empty page.

**Comment by MarcSkovMadsen** [comment]
Good idea

**Pull request merged**